contracts/EmptyContract.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.4.24;
+
+import "tidbit-eth/contracts/Oracles/BasicOracle.sol";
+import "tidbit-eth/contracts/Oracles/MedianOracle.sol";
+
+contract EmptyContract {
```

Comment:
This is a workaround to get contracts from tidbit-eth into the build folder. Will follow up to see if there's a better way.